enumerable_methods.rb

```diff
@@ -0,0 +1,122 @@
+# rubocop:disable all
+
```

Comment:
- [ ] 🛑 do not disable rubocop - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

  remove this line and fix the issues please.